view.js

```diff
@@ -12,24 +12,24 @@ let $toggleAll;
 let $newTodo;
 
 export function initView() {
-	$todoList = qs('.todo-list');
-	$todoItemCounter = qs('.todo-count');
-	$clearCompleted = qs('.clear-completed');
-	$main = qs('.main');
-	$toggleAll = qs('.toggle-all');
-	$newTodo = qs('.new-todo');
-	$delegate($todoList, 'li label', 'dblclick', ({ target }) => {
+	$todoList = qs(".todo-list");
+	$todoItemCounter = qs(".todo-count");
+	$clearCompleted = qs(".clear-completed");
+	$main = qs(".main");
+	$toggleAll = qs(".toggle-all");
+	$newTodo = qs(".new-todo");
+	$delegate($todoList, "li label", "dblclick", ({ target }) => {
 		editItem(target);
 	});
 }
 
 export function editItem(target) {
 	const listItem = target.parentElement.parentElement;
 
-	listItem.classList.add('editing');
+	listItem.classList.add("editing");
 
-	const input = document.createElement('input');
-	input.className = 'edit';
+	const input = window.document.createElement("input");
+	input.className = "edit";
```

Comment:
let's revert all quote changes in the JS files, so that it matches the original as much as possible.

Reply:
Gotcha. That was a ReSharper suggestion.